<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmoothPlasticSurfaceWidget</class>
 <widget class="QWidget" name="SmoothPlasticSurfaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="diffuse_reflectance_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Diffuse Reflectance</string>
   </property>
   <widget class="QFrame" name="diffuse_reflectance_frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="interior_refractive_index_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Interior refractive index</string>
   </property>
   <widget class="QFrame" name="interior_refractive_index_frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="exterior_refractive_index_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Exterior refractive index</string>
   </property>
   <widget class="QFrame" name="exterior_refractive_index_frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="specular_weight_box">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Specular weight</string>
   </property>
   <widget class="QSpinBox" name="specular_weight_spinbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="keyboardTracking">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000000</number>
    </property>
    <property name="value">
     <number>400</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
